frontend/app/src/modules/history/management/forms/OnlineHistoryEventForm.vue

```diff
@@ -250,6 +250,7 @@ defineExpose({
       ref="assetPriceForm"
       v-model:asset="asset"
       v-model:amount="amount"
+      :location="location"
```

Comment:
Question, why do we only do this for `OnlineHistoryEvent`, but not for the other (such as Evm Event)

Reply:
good catch, it's because I forgot to change location to required...

Reply:
I have adjusted it now @lukicenturi